docs/client/tracing.md

```diff
@@ -0,0 +1,84 @@
+# <img align="center" src="../images/logo.png">  Tracing
+
+Our generated code natively [`OpenCensus`][open_census] and [`OpenTelemetry`][open_telemetry]. To do so, generate with flag `--trace` (see our [flag index][flag_index] for more information).
+By default, all libraries log with a NoOpTracer that takes no action. To change this, you have to use setTracer to set a new default Tracer.
+
+To trace, make sure you install [our tracing library][tracing_library]:
+
+```
+npm install @azure/core-tracing
+```
+
+## OpenCensus
+
+Our generated SDKs handle retrieving context for you, so there's no need to pass in any context. Additionally, the
+OpenCensus threading plugin is included when installing this package.
+
+Since there is no explicit context you need to pass, you can create your usual OpenCensus tracer and call the generated SDKs.
+The following example uses the [Zipkin][zipkin] exporter.
+
+```js
+const tracing = require("@opencensus/nodejs");
+const { ZipkinTraceExporter } = require("@opencensus/exporter-zipkin");
+const {
+  setTracer,
+  OpenCensusTracerWrapper,
+  OpenCensusSpanWrapper
+} = require("@azure/core-tracing");
```

Comment:
Since in the other samples you have typescript I'd recommend standardizing to the same kind of imports
```suggestion
import tracing from "@opencensus/nodejs");
import { ZipkinTraceExporter } from "@opencensus/exporter-zipkin";
import {
  setTracer,
  OpenCensusTracerWrapper,
  OpenCensusSpanWrapper
} from "@azure/core-tracing";
```